pom.xml

```diff
@@ -29,7 +29,7 @@
     <version.org.mortbay.jetty>6.1.25</version.org.mortbay.jetty>
     <version.mysql>5.1.35</version.mysql>
     <version.postgresql>9.1-901-1.jdbc4</version.postgresql>
-    <version.com.fasterxml.jackson.core>2.7.2</version.com.fasterxml.jackson.core>
+    <version.com.fasterxml.jackson.core>2.10.0.pr1</version.com.fasterxml.jackson.core>
```

Comment:
i guess we need to change this in line what we have in our upstream master now. see https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1030/files

Reply:
DONE!